### ISSUE TITLE
<Select /> choices are not extracted correctly

### ISSUE THREAD

**Issue by chuyik** [lingui/js-lingui#919]
**Describe the bug**
Some choices on `<Select />` macro are missing after `lingui extract` called.

**To Reproduce**
```jsx
import { Select } from "@lingui/macro"

export default function App() {
  const gender = "female";
  return (
    <Select
      value={gender}
      male="His book"
      female="Her book"
      other="Their books"
    />
  );
}
```

After extraction, `male` and `female` missing:

```json
{
  "{gender, select, other {Their books}}": {
    "translation": "{gender, select, other {Their books}}",
    "extractedComments": []
  }
}
```

**Expected behavior**
```json
{
  "{gender, select, male {His book} female {Her book} other {Their books}}": {
    "translation": "{gender, select, male {His book} female {Her book} other {Their books}}",
    "extractedComments": []
  }
}
```

**Additional context**

- jsLingui version `3.3.0`
- Babel version
```bash
├─ @babel/core@7.12.10
├─ @jest/transform@26.6.2
│  └─ @babel/core@7.9.0
├─ babel-preset-react-app@10.0.0
│  └─ @babel/core@7.12.3
├─ jest-config@26.6.3
│  └─ @babel/core@7.9.0
└─ react-scripts@4.0.1
   └─ @babel/core@7.12.3
```
- Framework: Create React App

**Comment by chuyik** [comment]
However, if I use the `select` macro(not `Select`), it outputs as expected.

Dig into source code, I found that `Select` will detect if option matches plural rules, while `select` does not.

`select`: https://github.com/lingui/js-lingui/blob/23b06b5dfb/packages/macro/src/macroJs.ts#L318-L329
`Select`: https://github.com/lingui/js-lingui/blob/23b06b5dfb/packages/macro/src/macroJsx.ts#L296-L304

**Comment by semoal** [comment]
Released 3.5.0
